sdk/openai/openai/package.json

```diff
@@ -44,12 +44,12 @@
     "generate": "tsp-client update",
     "customize": "rimraf src && dev-tool customization apply -s sources/generated/src && npm run format",
     "bundle": "tsc -p . && dev-tool run bundle",
-    "build": "npm run clean && npm run bundle && api-extractor run --local",
+    "build": "npm run clean && npm run bundle && npm run extract-api",
```

Comment:
should we create another extract-api script that doesn't run the compiler since it already ran as part of the bundle one?

Reply:
hm good catch, we can call `dev-tool run extract-api` directly instead - I can push that up shortly